src/apps/loan-list/materials/stackable-material/material-info.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { FC, ReactNode } from "react";
+import { getAuthorNames, getMaterialInfo } from "../../utils/helpers";
+import { LoanMetaDataType } from "../../../../core/utils/helpers/LoanMetaDataType";
+import { GetMaterialManifestationQuery } from "../../../../core/dbc-gateway/generated/graphql";
+import { useText } from "../../../../core/utils/text";
+import { Cover } from "../../../../components/cover/cover";
+import { Pid } from "../../../../core/utils/types/ids";
+
+interface MaterialInfoProps {
+  loanMetaData: LoanMetaDataType;
+  material: GetMaterialManifestationQuery;
+  children: ReactNode;
+}
+
+const MaterialInfo: FC<MaterialInfoProps> = ({
+  loanMetaData,
+  material,
+  children
+}) => {
+  const t = useText();
+  const { creators, materialType, year, materialTitle, pid, description } =
+    getMaterialInfo(loanMetaData, material);
+
+  return (
+    <div className="list-reservation__material">
+      <div>
+        <Cover
+          pid={pid as Pid}
```

Comment:
I think the `pid` coming from `getMaterialInfo()` should be of the `Pid` type that we have globally.
Then you would not have to tranform it like this-